src/Sulu/Bundle/AdminBundle/Resources/js/containers/Form/tests/fields/Selection.test.js

```diff
@@ -557,7 +625,71 @@ test('Should pass props correctly to list component', () => {
     }));
 });
 
-test('Should pass resourceKey as listKey to list component if no listKey is given', () => {
+test('Should pass correct parameters to listStore', () => {
+    const value = [1, 6, 8];
+
+    const fieldTypeOptions = {
+        default_type: 'list',
+        resource_key: 'snippets',
+        types: {
+            list: {
+                adapter: 'table',
+                list_key: 'snippets_list',
+            },
+        },
+    };
+
+    const schemaOptions = {
+        request_parameters: {
+            value: [
+                {
+                    name: 'staticKey',
+                    value: 'some-static-value',
+                },
+            ],
+        },
+        resource_store_properties_to_request: {
+            value: [
+                {
+                    name: 'dynamicKey',
+                    value: 'otherPropertyName',
+                },
+            ],
+        },
+    };
+
+    const locale = observable.box('en');
+
+    const formInspector = new FormInspector(
+        new ResourceFormStore(
+            new ResourceStore('pages', 1, {locale}),
+            'pages'
+        )
+    );
+    formInspector.getValueByPath.mockReturnValue('value-returned-by-form-inspector');
```

Comment:
I think we should either implement the mock in a way that this value is only returned if `otherPropertyName` is passed as argument, or just check afterwards if the mock was actually called with that value.